src/client.cpp

```diff
@@ -876,18 +879,13 @@ void CClient::Start()
 
 void CClient::Stop()
 {
-    // stop audio interface
-    Sound.Stop();
+    // start disconnection
+    // Channel.Disconnect() should automatically disable Channel as soon as disconnected.
+    // Note that this only works if sound is active!
```

Comment:
So why make the changes here?

Reply:
I believe it's mainly moving around the Channel.Disconnect() and Sound.Stop() code around.